.dockstore.yml

```diff
@@ -129,6 +129,10 @@ workflows:
     primaryDescriptorPath: /pipes/WDL/workflows/detect_cross_contamination_precalled_vcfs.wdl
     testParameterFiles:
       - empty.json
+  - name: read_depths
+    subclass: WDL
+    primaryDescriptorPath: /pipes/WDL/workflows/read_depths.wdl
+    testParameterFiles:
```

Comment:
I think for some reason wants `testParameterFiles` to be non-empty, so they ask us to put a `- empty.json` entry in there even if that file doesn't exist

Reply:
Oh interesting. I copy/pasted another workflow, so this is not the only workflow to make this mistake.

Reply:
Nevermind, that was a lie. I will fix that. Thank you!

Reply:
I also noticed that the file was alphabetical before I got to it. It is alphabetical again.